docs/cloud-native-security/faq.asciidoc

```diff
@@ -1,15 +1,49 @@
+[[security-posture-faq]]
+= Frequently asked questions (FAQ)
+
+[[cspm-faq]]
+[discrete]
+== CSPM FAQ
+Frequently asked questions about the Cloud Security Posture Management (CSPM) integration and features.
+
+*Which cloud service providers are supported?*
+
+AWS is currently supported. Support for GCP & Azure is coming soon.
+
+*How often is my cloud security posture evaluated?*
+
+Cloud accounts are evaluated when you first deploy the CSPM integration and every 24 hours afterward.
+
+*Can I onboard multiple AWS accounts for posture monitoring?*
+
+Yes, you can monitor more than one cloud account. <<cspm-get-started, Add the CSPM integration>> again for each additional account you want to monitor.
+
+*Can I onboard multiple accounts via AWS organizations?*
+
+This capability is not currently supported but will be added in future iterations.
+
+*When do newly-enrolled cloud accounts appear on the dashboard?*
+
+After you deploy the CSPM integration, it can take up to 10 minutes for resource fetching, evaluation, and data processing before a newly-enrolled account appears on the Cloud Posture dashboard.
```

Comment:
```suggestion
After you deploy the CSPM integration, it can take up to 10 minutes for resource fetching, evaluation, and data processing before a newly enrolled account appears on the Cloud Posture dashboard.
```